README.md

```diff
@@ -36,35 +47,72 @@ The application has the following runtime dependencies:
+
+Redirect URL - `/customs/payment-records`
+
+| Enrolment Key	| Identifier Name | Identifier Value |
+| -------- | ------- | ------- | ------- | 
+| `HMRC-CUS-ORG` | `EORINumber`| `GB744638982000` |
+| `HMRC-CUS-ORG` | `EORINumber`| `GB744638982001` |
+
+## Testing
+
+The minimum requirement for test coverage is 90%. Builds will fail when the project drops below this threshold.
 
-There is just one test source tree in the `test` folder. Use `sbt test` to run them.
+### Unit Tests
 
-To get a unit test coverage report, you can run `sbt clean coverage test coverageReport`,
-then open the resulting coverage report `target/scala-2.11/scoverage-report/index.html` in a web browser.
+| Command    | Description |
+| -------- | ------- |
+| `sbt test` | Runs unit tests locally |
+| `sbt "test:testOnly *TEST_FILE_NAME*"` | runs tests for a single file |
 
-Test coverage threshold is set at 90% - so if you commit any significant amount of implementation code without writing tests, you can expect the build to fail.
+### Coverage
+
+| Command    | Description |
+| -------- | ------- |
+| `sbt clean coverage test coverageReport` | Generates a unit test coverage report that you can find here target/scala-2.11/scoverage-report/index.html  |
+
+## Available Routes
+
+You can find a list of microservice specific routes here - `customs-financials-frontend/conf/app.routes`
+
+Application entrypoint:  `/customs/payment-records` 
 
 ## Feature Switches
 
-Feature switches can be enabled per-environment via the `app-config-<env>` project:
+> ### Caution!
+> There's a risk of WIP features being exposed in production! 
+> **Don't** enable features in `application.conf`, as this will apply globally by default
+
+### Enable features
+| Command    | Description |
+| -------- | ------- |
+| `sbt "run -Dfeatures.some-feature-name=true"` | enables a feature locally without risking exposure |
+
+### feature flag examples
+| Flag    | Description |
+| -------- | ------- |
+| `fixed-system-time` | Enable the fixed datetime for DateTimeService by enabling the stub data for current and requested statements. It is only enabled in Development env |
+| `xi-eori-enabaled` | Enable the XI EORI related api calls |
+| `cash-account-v2-enabled` | Enable the new cash account design |
+| `home-page-links-enabled` | Control the display of 'Account authorities' and 'Find accounts you have authority to use' links. Must be false for all the env |
+| `authorities-notification-panel-enabled` | Enable display of notifications panel |
+| `eu-eori-enabled` | Enable the EU EORI feature |
 
-    ...
-    features.some-feature: true
-    ...
+fixed-system-time - flag to enable the fixed datetime for DateTimeService. It is used to enable the stub data for current and requested statements. It is only enabled in Development env
```

Comment:
Seems, the contents from line 107-112 is repeat of lines 100-105. Could also be seen in Preview mode. Could you please remove this duplication?